expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -129,14 +129,17 @@ public ExpandedDataEntry expandEntry(Entity dataEntry) throws JsonProcessingExce
     }
 
     private ObjectNode replaceInlineContextWithUriContext(String resourceWithNviValues) {
-        var objectNode =  attempt(() -> (ObjectNode) objectMapper.readTree(resourceWithNviValues)).orElseThrow();
-        if (!objectNode.at(CONTEXT_NODE_SELECTOR).isMissingNode()) {
-            objectNode.remove(CONTEXT_FIELD_PROPERTY_NAME);
+        var objectNode = attempt(() -> (ObjectNode) objectMapper.readTree(resourceWithNviValues)).orElseThrow();
+        if (hasContextNode(objectNode)) {
```

Comment:
do we even need to check? Cant we just always put the context uri?

Reply:
In tests, this isn't always the case alas